src/plugin.py

```diff
@@ -84,6 +84,7 @@ def __init__(self, reader, writer, token):
         self._achievements_cache = Cache()
         self._achievements_cache_updated = False
         self._achievements_semaphore = asyncio.Semaphore(20)
+        self._tick_count = 0
 
         self.create_task(self._update_local_games(), "Update local games")
```

Comment:
Instead of

```
self._tick_count = 0
self.create_task(self._update_local_games(), "Update local games")
```
do
 `
self._local_games_update_task = self.create_task(self._update_local_games(), "Update local games")`